docker/datahub-mae-consumer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Defining environment
 ARG APP_ENV=prod
 
-FROM --platform=$BUILDPLATFORM alpine:3 AS base
```

Comment:
Not familiar with what this --platform flag is used for.. I guess not relevant?

Reply:
I believe this is the cause of our broken arm images.